### ISSUE TITLE
To move a full notebook into a different notebook 

### ISSUE THREAD

**Issue by Ypot** [orgzly/orgzly-android#379]
Sometimes you can create a notebook that seems OK. 
But, after that, you feel like you must create a different notebook more general than the first one.
In that case, each note from the first created notebook must be moved, one by one. 
To be able to move a full notebook into another one, would soar the flexibility with which the app can be used.

Example:

Notebook: orgzly
* note 1
** note 1.1
* note 2
* note 3

New Notebook: open source software
* emacs
** orgmode
(* orgzly) - > this note, and its sub-notes, would be generated just by moving the "orgzly notebook". Now, instead, a new note called must be made and its subnotes (note 1, note 2, note n) moved one by one. 

Thanks for your great work

**Comment by Ypot** [comment]
Hi

I hope you are right, but reading again and again those issues, I can't see any references to move a complete notebook into a different notebook.

As said, I hope you got me, and to see this feature soon.
:-)

**Comment by Ypot** [comment]
Thanks nevenz. But the new note 
./orgzly 
would be automatically created when you refile all of its subnotes? 

I was even thinking about moving one notebook into another one, while you are in the screen of the notebooks, being unnecessary to go into each notebook you want to refile

**Comment by nevenz** [comment]
> But the new note ./orgzly would be automatically created when you refile all of its subnotes?

Yes, refiling UI would let you select the existing note, but can also allow optionally inputting one title in addition which would be created.

> I was even thinking about moving one notebook into another one, while you are in the screen of the notebooks, being unnecessary to go into each notebook you want to refile

I guess the same refilling UI could be used and support refilling entire notebooks too. It can be done as part of #339.
